src/main/resources/reference.conf

```diff
@@ -46,4 +46,26 @@ videobridge {
     # Must be set when enabled = true
     # domain="some-domain"
   }
+  ice {
+    tcp {
+      # Whether ICE/TCP is enabled.
+      enabled = false
+      port = 443
+      # Port to fallback to in case binding to 'port' fails. Set to -1 to disable fallback.
+      fallback-port = 4443
+      # An optional additional port to advertise.
+      # mapped-port = 8443
+      # Whether to use "ssltcp" or plain "tcp".
+      ssltcp = true
+    }
+    # The port for ICE/UDP.
+    port = 10000
+    # An optional prefix to include in STUN username fragments generated by the bridge.
+    # ufrag-prefix = "jvb-123:"
+
+    # Which candidate pairs to keep alive. One of "selected_and_tcp", "selected_only", or "all_succeeded".
```

Comment:
There's an enum for this, right? Should we refer to it here?

Reply:
If you are a user reading the config file the raw values are more useful than the class name.

Reply:
Sure, it's just also more likely to get out of date.  Not opposed to keeping the strings here, but referring to the enum name would be nice.